examples/gno.land/r/matijamarjanovic/home/config.gno

```diff
@@ -0,0 +1,64 @@
+package home
+
+import (
+	"errors"
+	"std"
+)
+
+var (
+	mainAddr   = std.Address("g1ej0qca5ptsw9kfr64ey8jvfy9eacga6mpj2z0y") // matija's main address
+	backupAddr std.Address                                               // backup address
+
+	errorInvalidAddr  = errors.New("config: invalid address")
+	errorUnauthorized = errors.New("config: unauthorized")
+)
+
+func Address() std.Address {
+	return mainAddr
+}
+
+func Backup() std.Address {
+	return backupAddr
+}
+
+func SetAddress(newAddress std.Address) error {
+	if !newAddress.IsValid() {
+		return errorInvalidAddr
```

Comment:
note: it is idiomatic in gno to panic in these cases, as panicking rolls back the transaction, which is better here